app/templates/frameworks/_guidance_links.html

```diff
@@ -1,87 +1,104 @@
-<li class="browse-list-item">
-{% if framework.status in ['standstill', 'live'] and countersigned_agreement_file %}
-  <h2>
-    Documents, updates and questions
-  </h2>
-  <ul class="list-no-bullet">
-    <li>
-        <a href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name=countersigned_agreement_file) }}">
-        <span>Download your countersigned framework agreement (.pdf)</span>
-        </a>
-    </li>
+<div class="dmspeak">
+  <div>
+    <h2 class="heading-xmedium">Guidance</h2>
 
-    <li>
-        <a href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name=result_letter_filename) }}">
-        <span>Download your application result letter (.pdf)</span>
-        </a>
-    </li>
+    {% if communications_files.invitation.last_modified %}
+    <p>
+      <a href="{{ url_for('.download_supplier_file', filepath=communications_files.invitation.filename, framework_slug=framework.slug) }}">
+        Download the invitation to apply
+      </a>
+    </p>
+    {% endif %}
 
-    <li>
-        <a href="{{ url_for('.download_supplier_file', filepath=supplier_pack_filename, framework_slug=framework.slug) }}">
-        <span>Download guidance and legal documentation (.zip)</span>
-        </a>
-        {% if last_modified.supplier_pack %}
-        <div class="hint">
-        Last updated
-        <time datetime="{{ last_modified.supplier_pack }}">
-            {{ last_modified.supplier_pack|dateformat }}
-        </time>
-        </div>
+    <p>
+      <a href="{{ framework_urls.supplier_guide_url }}">
+        Read about how to 
+        {% if supplier_framework.agreementReturned %}
+          sell your services
+        {% else %}
+          apply
         {% endif %}
-    </li>
+      </a>
+    </p>
+  </div>
+{% if framework.status not in ['standstill', 'live'] and not supplier_framework.agreementReturned %}
```

Comment:
I don't think we need the `and` part of this clause because `supplier_framework.agreementReturned` can only be set while the framework is `in ['standstill', 'live']`, so if framework isn't standstill or live then there's no way the agreement can be returned.